src/plugin/cursus/Controller/EventPresenceController.php

```diff
@@ -126,6 +129,73 @@ public function updateStatusAction(string $status, Request $request): JsonRespon
         }, $presences));
     }
 
+    /**
+     * Updates the status of a EventPresence for current user.
+     *
+     * @Route("/sign", name="apiv2_cursus_event_presence_sign", methods={"PUT"})
+     */
+    public function signStatusAction(Request $request): JsonResponse
+    {
+        $data = $this->decodeRequest($request);
+        if (empty($data)) {
+            return new JsonResponse(null, 404);
```

Comment:
C'est une erreur dans la requête, ce devrait être une erreur 400 Bad request, pas une 404 not found.
Dans ce genre de cas tu peux juste throw une exception InvalidDataException (symfony va la convertir automatiquement en erreur 400).